src/legacy/server/i18n/get_kibana_translation_paths.test.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { I18N_RC } from './constants';
+import { fromRoot } from '../../../core/server/utils';
+
+jest.mock('./get_translation_paths', () => ({ getTranslationPaths: jest.fn() }));
+import { getKibanaTranslatoinPaths } from './get_kibana_translation_paths';
```

Comment:
nit: Typo/spelling of "...Translatoin..." -> "...Translation..."
also on lines 36 and 50